client/src/Components/SingularPost.js

```diff
@@ -21,8 +24,66 @@ export default function SingularPost(props) {
     setOpen(false);
   };
 
+  const handlePostLikeClick = () => {
+    if (postLikedBySelf) {
+      setPostLikedBySelf(false);
+      setPostLikes(postLikes - 1); // cosmetic
+      // delete like
+      authFetch(
```

Comment:
Maybe it's better to modify the state only on a successful response

Reply:
Likely no maybe about it.  Changed accordingly.